### ISSUE TITLE
asyncio.Task.all_tasks has been removed in Python 3.9

### ISSUE THREAD

**Issue by tirkarthi** [aio-libs/aiosmtpd#188]
https://github.com/aio-libs/aiosmtpd/blob/ec804d9038989b20d2068ab3dc69ff0888a48257/aiosmtpd/controller.py#L64

Ref : https://github.com/python/cpython/pull/20874

**Comment by moggers87** [comment]
The fix is quite simple:

```
try:
    _all_tasks = asyncio.all_tasks
except AttributeError:
    # For Python < 3.7
    _all_tasks = asyncio.Task.all_tasks
```

I did have a PR open with this patch, but Python 3.9 hasn't been released yet.

**Comment by tirkarthi** [comment]
Python 3.9 will have it's last beta release by this month. The final release schedule is at https://www.python.org/dev/peps/pep-0596/

**Comment by pepoluan** [comment]
Python 3.9.0 is released. Maybe time to revisit this?

**Comment by moggers87** [comment]
It's not available on Travis just yet.

**Comment by pepoluan** [comment]
> It's not available on Travis just yet.

True, but we can already test against 3.9 on our systems. On Linuxes, with the help of `pyenv` I've successfully tested against 3.9

**Comment by pepoluan** [comment]
This has been implemented when we merged #192, so I'm closing this issue.

Feel free to reopen if necessary.
